### PR TITLE
Add some example queries that use regex

### PR DESCRIPTION
**Pull request by mammykins**

### CODE REVIEW

File: src/neo4j/fav_queries.cypher [hunk 1]
```diff
@@ -59,3 +59,17 @@ MATCH (p:Person)-[k:HAS_ROLE]->(:Role)-[]-(:Organisation {name: 'Home Office'})<
 WHERE p.name CONTAINS 'Theresa May' AND (k.startDate.year > 2009 AND k.endDate.year < 2017)
 RETURN c.contentID, c.name
 LIMIT 25;
+
+// Find Cid's with a specific word or pattern using regular expressions (consider literal match first).
+MATCH (n:Cid)-[:IS_TAGGED_TO]->(t:Taxon {name:'Business and industry'})
+WHERE n.title =~ '(?i).*\\bBREXIT\\b.*' AND n.documentType = "['guidance']"
```
Comment (ff-l) [inline src/neo4j/fav_queries.cypher]: Can you remind me why the `document_type` is a list (or a string with square brackets) ?

File: src/neo4j/fav_queries.cypher [hunk 2]
```diff
@@ -61,12 +61,13 @@ RETURN c.contentID, c.name
 LIMIT 25;
 
 // Find Cid's with a specific word or pattern using regular expressions (consider literal match first).
+// Note: You need to backslash backslashes.
```
Comment (ff-l) [inline src/neo4j/fav_queries.cypher]: also, I'm not sure we ought to be documenting how tools/libs, here regexes work. Backslashing literals is standard procedure.

File: src/neo4j/fav_queries.cypher (hunk 1 above)
Comment (mammykins) [reply src/neo4j/fav_queries.cypher]: tldr: `content_id` one-to many `base_path` or
`content_id` ⋲`base_path`

The issue is that we start with a list of one `base_path` per row (each with their own `document_type`). In the knowledge graph we want to represent `content_id` as the unit of interest, thus we collapse to one `content_id` per row. As there are more than one `base_path` and `document_type` for some `content_id` we store as a list. I believe the Content Data team came up with a solution for this problem that we could investigate later after prototyping.

Comment (ff-l) [reply src/neo4j/fav_queries.cypher]: Eh... could we just take the first `base_path` and/or drop duplicates in advance? The lists aren't looking good

**Pull request merged**